Resources/Prototypes/Reagents/medicine.yml

```diff
@@ -697,6 +697,7 @@
   desc: reagent-desc-cognizine
   group: Medicine
   physicalDesc: reagent-physical-desc-enigmatic
+  flavor: magical
```

Comment:
This is rare enough you can probably do something funny here. Maybe it should taste "like a nightmare for philosophers" or something.

Reply:
oh fuck i didn't think it would come up as a review woops

Reply:
I did think about putting in a joke taste but decided to use a preexisting taste instead of making a new one. It would be funny though, maybe I should do it